src/common/njord.ts

```diff
@@ -644,6 +644,53 @@ export const awardPost = async (
           )
           .execute();
 
+        let newComment: Comment | undefined;
```

Comment:
@capJavert Sort of duplication, but it needed the different order compared to the one on comments :jenkins-triggered:

Reply:
All good